CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * BREAKING cli binary renamed from argo-cloudops to cello
 * Environment variables prefix changed from ARGO_CLOUDOPS to CELLO (backwards compatible)
 
+## [0.13] - 2022-05-02
```

Comment:
changelog looks outdated. 0.13.0 is already released and the version needs to be semver format.

https://github.com/cello-proj/cello/blob/main/CHANGELOG.md

Reply:
done, 0.13.1 (patch) should be fine compared to 0.13.0, eg. this shouldn't be any more breaking than the prev change